playbook-website/app/javascript/components/MainSidebar/NavComponents/KitsNavComponent.tsx

```diff
@@ -56,14 +57,14 @@ export const KitsNavItem = ({
   };
 
   //click on nested items
-  const handleSubItemClick = (j, sublink, parentIndex) => {
+  const handleSubItemClick = (j, sublink, Index) => {
```

Comment:
Can we avoid single letter param names? Let's make it more obvious what `j` is.

Reply:
sounds good! on it